docs/bbs/maps.md

```diff
@@ -6,4 +6,405 @@ Maps in BBS consist of an [ARC](.\arc.md) file which is loaded into memory in it
+|  YT04   | 塔:魔法使いの部屋 | Sorcerer's Chamber   |             |
+
+**Radiant Garden**
+
+| **Map** | **Japanese** | **English** | **Comment** |
+|---------|--------------|-------------|-------------|
+|  RG00   | レイディアントガーデン | Radiant Garden  |             |
+|  RG01   | 城外庭園 | Outer Garden  |             |
+|  RG02   | エントランス | Entryway  |             |
+|  RG03   | 中央広場 | Central Square  |             |
+|  RG04   | 水路 | Aqueduct  |             |
+|  RG05   | 住宅街 | Castle Town  |             |
+|  RG06   | 動力炉 | Reactor  |             |
+|  RG07   | 噴水広場 | Fountain Court  |             |
+|  RG08   | マーリンの家 | Merlin's House  |             |
+|  RG09   | 庭園 | Gardens  |             |
+|  RG10   | 正面扉口 | Front Doors  |             |
+|  RG11   | 浄化施設 | Purification Facility  |             |
+|  RG12   | 城外庭園 | Outer Gardens  |             |
+|  RG13   | 中央広場 | Central Square  |             |
+|  RG14   | 中央広場 | Central Square  |             |
+
+**FOR EVENTS ONLY**
+
+| **Map** | **Japanese** | **English** | **Translation** | **Comment** |
+|---------|--------------|-------------|-------------|--------
+|  JB00   | イベント専用 | FOR EVENTS ONLY  |             | JP Version Only
+|  JB01   | ルーイの遺跡:広場 | Louie's Ruins  |             | JP Version Only
+|  JB02   | ルーイの遺跡:通路 | Court  |             | JP Version Only
+|  JB03   | 分岐点 | Path Crossroads  |             | JP Version Only
+|  JB04   | 地下遺跡:入口 | UG Ruins Entrance  |             | JP Version Only
+|  JB05   | 地下遺跡:通路1 | UG Ruins Passage 1   |             | JP Version Only
+|  JB06   | 地下遺跡:通路2 | UG Ruins Passage 2 |             | JP Version Only
+|  JB07   | 地下遺跡:広場 | UG Courtyard  |             | JP Version Only
+|  JB08   | 遺跡隣接ジャングル | Jungle Near Ruins  |             | JP Version Only
+|  JB09   | 高台 | Eminence  |             | JP Version Only
+|  JB10   | ジャングル | Jungle  |             | JP Version Only
+|  JB11   | 人間の村近くの川 | Man-Village River  |          | JP Version Only
+|  JB12   | 湿地 | Bog  |             | JP Version Only
+|  JB20   | 闇の世界：中層 | Realm of Darkness: Middle Zone  |             | Final Mix
+|  JB21   | 闇の世界：下層 | Realm of Darkness: Lower Zone  |             | Final Mix
+|  JB22   | 闇の世界：上層 | Realm of Darkness: Upper Zone  |             | Final Mix
+|  JB23   | 闇の狭間 | The Dark Margin  |             | Final Mix
+|  JB24   | 闇シンデレラ城（FM） |   |  Cinderella's Castle (FM)         | Final Mix
+|  JB30   | デスティニーアイランド（ひる） |   | Destiny Islands (Noon)     | Text only present in HD ReMiX
+|  JB31   | トラヴァース２番街 |   | Traverse Town 2nd District            | Text only present in HD ReMiX
+|  JB32   |  |   |             | Traverse Town 1st District
+|  JB33   |  |   | Destiny Islands (Afternoon)            | Text only present in HD ReMiX
+|  JB34   |  |   | Destiny Islands (Night)            | Text only present in HD ReMiX
+|  JB35   | トラヴァース３番街 |   | Traverse Town 3rd District            | Text only present in HD ReMiX
+|  JB50   | 異空の回廊 | Lanes Between  |             |
```

Comment:
Does JB50 changes between the game versions? For Vanilla JP it seems to be Lanes Between

Reply:
JB50 has always been Lanes Between as is mentioned in this list.